### ISSUE TITLE
Module error

### ISSUE THREAD

**Issue by icepick3000** [cooldevil007/alexa-VideoDemo#1]
I copied your code to the alexa code builder but I get this error; 

 "errorMessage": "Cannot find module 'alexa-sdk'",

I noticed you included modules in your project.. but how do I upload then to the alexa code builder?

**Comment by cooldevil007** [comment]
I have used Alexa CLI to create a template project and from there I have
change the default template code with my own code. Video and audio skill
can’t be created directly in lambda editor

**Comment by icepick3000** [comment]
Okay I just installed the Alexa CLI. Is it just a matter of downloading your project as a zipfile and running a deploy?

**Comment by cooldevil007** [comment]
Yes just clone my project and after that you have to run deploy but
remember your ask should be configured with AWS account as well otherwise
it won’t deploy your lambda code

**Comment by icepick3000** [comment]
Somewhere I am going wrong on this.. 
I did the following steps.. 

Downloaded your project as a zip file
Unzipped it and cd't into the directory
I run 'ask deploy' 

I get this error;
Error code: 401
{
  "message": "User is not authorized to access this skill."
}

My ask init shows this (I setup the user for AWS);

![screen shot 2018-02-12 at 19 17 18](https://user-images.githubusercontent.com/36381026/36112269-68aba456-1029-11e8-9a7f-8e448b14328a.png)

Any ideas what I am doing wrong?
